### ISSUE TITLE
Dealias symbols

### ISSUE THREAD

**Issue by olafurpg** [scalameta/scalameta#863]
Currently, in scalafix, when rewriting for example the stdlib `Left` it's necessary to handle both the symbol `scala.util.Left` and `scala.package.Left`. It would be nice if it was possible to dealias appearances of `scala.package.Left` with `scala.util.Left` with some means in the semantic API, for example denotations. Currently, `scala.package.Left` has no denotations unless we build the standard library with scalahost to build a semanticdb.

**Comment by olafurpg** [comment]
I made a small experiment to see if current denotations can help in this situation and it appears that this issue is blocked by #822 .

```scala
  type Foo[A, B] = scala.util.Left[A, B]
  // ...
  Foo.symbol.denot.into // "[A, B]scala.util.Left[A,B]"

**Comment by olafurpg** [comment]
I propose to push this back one milestone. It's not urgent for scalafix.

**Comment by xeno-by** [comment]
This is fixed in 3.3.0 via `SymbolInformation.tpe`. See the specification to learn what signatures are emitted for type aliases.
